README.md

```diff
@@ -31,14 +35,35 @@ unsubscribe('b', h)
 ```
 
 ## Naming convention
+
 To stay consistent, we encourage you to use the following syntax when declaring events
 
 `app-id:object:verb`
 
 ### Examples:
-- nextcloud:unified-search:closed
-- files:file:uploading
-- files:file:uploaded
-- contacts:contact:deleted
-- calendar:event:created
-- forms:answer:updated
+
+- `nextcloud:unified-search:closed`
+- `files:file:uploading`
+- `files:file:uploaded`
+- `contacts:contact:deleted`
+- `calendar:event:created`
+- `forms:answer:updated`
+
+## Development
+
+```sh
+npm install
+
+npm run build
+npm run test
+```
+
+### Requirements
+
+- [Node 14 or higher](https://nodejs.org/en/download/)
+- [NPM 7 or higher](https://www.npmjs.com/package/npm)
```

Comment:
```suggestion
- [Node 16 or higher](https://nodejs.org/en/download/)
- [NPM 7 or higher](https://www.npmjs.com/package/npm)
```

We are using Node 16 now as a default: https://github.com/nextcloud/nextcloud-event-bus/commit/6f0429d39b926c3b3b648b9f45ecb7a22f32bb64